README.md

```diff
@@ -170,42 +170,42 @@ Golang import:
 import "gopkg.in/confluentinc/confluent-kafka-go.v1/kafka"
 ```
 
-librdkafka
+`librdkafka`
 ----------
 
-Prebuilt librdkafka binaries are included with the Go client and librdkafka
+Prebuilt `librdkafka` binaries are included with the Go client and `librdkafka`
 does not need to be installed separately on the build or target system.
-The following platforms are supported by the prebuilt librdkafka binaries:
+The following platforms are supported by the prebuilt `librdkafka` binaries:
 
  * Mac OSX x64
  * glibc-based Linux x64 (e.g., RedHat, Debian, CentOS, Ubuntu, etc) - without GSSAPI/Kerberos support
  - musl-based Linux 64 (Alpine) - without GSSAPI/Kerberos support
 
-When building your application for Alpine Linux (musl libc) you must pass
+When building your application for Alpine Linux (`musl libc`) you must pass
 `-tags musl` to `go get`, `go build`, etc.
 
-`CGO_ENABLED` must NOT be set to 0 since the Go client is based on the
-C library librdkafka.
+`CGO_ENABLED` must NOT be set to `0` since the Go client is based on the
+C library `librdkafka`.
 
 If GSSAPI/Kerberos authentication support is required you will need
-to install librdkafka separately, see the **Installing librdkafka** chapter
+to install `librdkafka` separately, see the **Installing librdkafka** chapter
 below, and then build your Go application with `-tags dynamic`.
 
 
 
-Installing librdkafka
+Installing `librdkafka`
 ---------------------
 
-If the bundled librdkafka build is not supported on your platform, or you
-need a librdkafka with GSSAPI/Kerberos support, you must install librdkafka
+If the bundled `librdkafka` build is not supported on your platform, or you
+need a `librdkafka` with GSSAPI/Kerberos support, you must install `librdkafka`
 manually on the build and target system using one of the following alternatives:
 
 - For Debian and Ubuntu based distros, install `librdkafka-dev` from the standard
 repositories or using [Confluent's Deb repository](http://docs.confluent.io/current/installation.html#installation-apt).
 - For Redhat based distros, install `librdkafka-devel` using [Confluent's YUM repository](http://docs.confluent.io/current/installation.html#rpm-packages-via-yum).
-- For MacOS X, install `librdkafka` from Homebrew. You may also need to brew install pkg-config if you don't already have it: `brew install librdkafka pkg-config`.
+- For MacOS X, install `librdkafka` from Homebrew. You may also need to brew install `pkg-config` if you don't already have it: `brew install librdkafka pkg-config`.
```

Comment:
Should probably be `brew install pkg-config`

Reply:
I think it's probably better as-is to head off anyone reading this thinking _"Oh, I only need to type `brew install pkg-config` and it'll take care of this line"_, not realizing they also need a `brew install librdkafka`. Better to default to the n00b case IMO.